tests/plugins/elementspath/manual/elementspath.html

```diff
@@ -0,0 +1,82 @@
+<textarea id="editor1">
+	<p>Hello world. <strong>Bold text and <em>italic with additional <u>underline</u></em></strong>.</p>
+	<p>Hello world. <strong contenteditable="false">Bold text and <em contenteditable="true">italic with additional <u>underline</u></em></strong>.</p>
+	<ul>
+		<li>One</li>
+		<li>Two<ol>
+			<li><strong>STRONG: </strong>1111</li>
+			<li><em>ITALIC: </em>2222</li>
+		</ol></li>
+	</ul>
+	<div><div><div><div><div><div><div><div><div><div><div>Divception</div></div></div></div></div></div></div></div></div></div></div>
```

Comment:
Nice idea, but it is hard to evaluate if path is correct without looking at the source. You may slightly decrese the number of divs (like 5) and change the text to e.g.

> Divception - when clicked elements path will show 5 divs.